core/src/integration-test/java/com/scalar/db/storage/cassandra/CassandraAdminTestUtils.java

```diff
@@ -25,7 +37,9 @@ public void corruptMetadata(String namespace, String table) {
   }
 
   @Override
-  public void dropNamespacesTable() throws Exception {
-    throw new UnsupportedOperationException("Not yet implemented");
+  public void dropNamespacesTable() {
+    String dropKeyspaceQuery =
+        SchemaBuilder.dropKeyspace(quoteIfNecessary(metadataKeyspace)).getQueryString();
+    clusterManager.getSession().execute(dropKeyspaceQuery);
```

Comment:
Don't we need to add the same to `MultiStorageAdminTestUtils`?

Reply:
Good catch, thank you. I revised it. Please have a look again.